test/plugins/test_permissions.py

```diff
@@ -36,6 +41,8 @@ def test_failing_to_set_permissions(self):
     def do_thing(self, expect_success):
         if platform.system() == "Windows":
             self.skipTest("permissions not available on Windows")
+        if is_wsl_and_ntfs(os.getcwd()):  # WSL on NTFS does not support chmod
```

Comment:
What does `platform` return for WSL? Isn't it `Windows`? If so, it seems that the check on line 37 already skips the test?